### PR TITLE
fix: destructured props for inline components

### PR DESCRIPTION
**Pull request by Varixo**
Convert destructured props for an inline component to single prop

### CODE REVIEW

**Review by wmertens — APPROVED** [review]
:muscle:

File: packages/qwik/src/optimizer/core/src/snapshots/qwik_core__test__example_lightweight_functional.snap [hunk 1]
```diff
@@ -9,7 +9,7 @@ snapshot_kind: text
 
 import { $, component$ } from '@qwik.dev/core';
 
-export const Foo = component$(({color}) => {
+export const Foo = component$((props) => {
```
Comment (wmertens) [inline packages/qwik/src/optimizer/core/src/snapshots/qwik_core__test__example_lightweight_functional.snap:12]: This one was wrong actually, maybe we want to use some props? Not sure if important

Comment (Varixo) [reply packages/qwik/src/optimizer/core/src/snapshots/qwik_core__test__example_lightweight_functional.snap:12]: What do you mean? Later we use `...props` and never the `color`, so I think now it is correct?
or do you mean that this error is intentional?

Comment (wmertens) [reply packages/qwik/src/optimizer/core/src/snapshots/qwik_core__test__example_lightweight_functional.snap:12]: No it was destructuring color but using props

Comment (Varixo) [reply packages/qwik/src/optimizer/core/src/snapshots/qwik_core__test__example_lightweight_functional.snap:12]: yeah, and it is wrong, right? or should I revert it?

**Pull request merged**